src/applications/vaos/tests/mocks/form.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { combineReducers, applyMiddleware, createStore } from 'redux';
+import thunk from 'redux-thunk';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import { fireEvent, waitFor } from '@testing-library/dom';
+
+import { commonReducer } from 'platform/startup/store';
+import { renderInReduxProvider } from 'platform/testing/unit/react-testing-library-helpers';
+
+import reducers from '../../reducers';
+import newAppointmentReducer from '../../reducers/newAppointment';
+
+import TypeOfCarePage from '../../containers/TypeOfCarePage';
+import { cleanup } from '@testing-library/react';
+
+export function createTestStore(initialState) {
+  return createStore(
+    combineReducers({
+      ...commonReducer,
+      ...reducers,
+      newAppointment: newAppointmentReducer,
+    }),
+    initialState,
+    applyMiddleware(thunk),
+  );
+}
+
+export async function setTypeOfCare(store, label) {
+  const router = {
+    push: sinon.spy(),
+  };
+  const { findByLabelText, getByText } = renderInReduxProvider(
+    <TypeOfCarePage router={router} />,
+    { store },
+  );
+
+  const radioButton = await findByLabelText(label);
+  fireEvent.click(radioButton);
+  fireEvent.click(getByText(/Continue/));
+  await waitFor(() => expect(router.push.called).to.be.true);
+  await cleanup();
+
+  return router.push.firstCall.args[0];
```

Comment:
Just curious what is returned here and how would it be used? I don't see the tests below using the return value.

Reply:
Yeah, I'm not sure if we would use actually use it. I returned it so that someone calling this could verify that it moved on to the correct page (since there are several different pages that could be next) without having to pass their own router mock in. But I didn't feel the need to check that in the tests I wrote, so maybe it won't be needed in the future, either.